packages/trader/src/exchange-adapters/UniswapTrader.ts

```diff
@@ -61,27 +57,6 @@ class UniswapTrader implements InstanceType<typeof ExchangeAdapterInterface> {
       return error;
     }
   }
-
-  async getExchangeSpotPrice() {
```

Comment:
deprecated this in favour of using the existing price feed

Reply:
The SA/English spelling of favor always sounds so sophisticated :)

Reply:
🎩